### ISSUE TITLE
Usernames being shuffled in comment section (possibly intermittent or resolved issue)

### ISSUE THREAD

**Issue by david0178418** [OdyseeTeam/odysee-frontend#1060]
See the image below [which was taken from this post](https://odysee.com/premium:10bcd8472e275ffab532e7f07dcfdcd05ce331b3?lc=1fe55c305535fd420969b2825dd9174eaa9cbbb184b10ed58ad6e64eaf23f80d)

It's possibly an intermittent or resolved issue as I cannot reproduce it again.  I've still got the window open for the time being, in case this is some Heisenbug you've been tracing down.  Otherwise, this might be considered just an "FYI" and ok to immediately close.

![image](https://user-images.githubusercontent.com/128739/157525600-b4cb5914-64d6-417a-9708-61aac3eabce2.png)

**Comment by tzarebczan** [comment]
Yep, we noticed that. Pushed something up a couple days ago that was supposed to help but it didn't. Thanks for filing!
